### PR TITLE
Fix errors based on feedback

### PR DESCRIPTION
**Pull request by femisystems**

### CODE REVIEW

File: src/js/app.js [hunk 1]
```diff
@@ -2,17 +2,13 @@
     .controller('MainController', function () {
       this.stringMethods = Object.keys(STRING_EXTENSION);
       this.response = '';
-      this.evaluate = function (inputString) {
-        const stringMethod = document.getElementById('stringMethod').value;
-        const display = document.getElementById('display');
-        let currentFunc = '';
+      this.evaluate = function () {
```
[the remark this answers is not in the document]
Comment (andela-kakpobome) [reply src/js/app.js]: ### Nitpick

You should use big arrow functions for anonymous functions instead.

[the remark this answers is not in the document]
Comment (andela-kakpobome) [reply src/js/app.js]: @andela-fabolaji this is currently blocking me from giving the full green light on this?

File: src/js/app.js [hunk 2]
```diff
@@ -1,18 +1,14 @@
-  angular.module('StringExtension', [])
-    .controller('MainController', function () {
-      this.stringMethods = Object.keys(STRING_EXTENSION);
-      this.response = '';
-      this.evaluate = function (inputString) {
-        const stringMethod = document.getElementById('stringMethod').value;
-        const display = document.getElementById('display');
-        let currentFunc = '';
+angular.module('StringExtension', [])
+  .controller('MainController',['$scope', ($scope) => {
```
[the remark this answers is not in the document]
Comment (andela-kakpobome) [reply src/js/app.js]: @andela-fabolaji you should listen to HoundCI on this issue.

[the remark this answers is not in the document]
Comment (femisystems) [reply src/js/app.js]: Will fix it now

**Pull request merged**